### PR TITLE
Images

### PR DESCRIPTION
**Pull request by killerclaws12**
Added screenshots of bot to README.md so people can get a idea of what the bot looks like, and what some of the commands look like.

### CODE REVIEW

**Review by virtual-designer — CHANGES_REQUESTED** [review]
Everything looks good, but the images are at the bottom, plus there are multiple. I recommend having one single screenshot.

**Comment by killerclaws12** [comment]
If theres one, people wont have an idea of what some of the commands look like. You can also make the images go side by side and make them smaller. >-<

**Comment by killerclaws12** [comment]
or make a drop down menu on the readme file

File: package.json [hunk 1]
```diff
@@ -1,95 +1,95 @@
...
-        "clean": "bash -c 'rm -r build'",
-        "prepare": "sh -c \"husky install; sh ./init.sh\" || echo \"Error while trying to prepare project.\"",
-        "start": "pm2 start ./ecosystem.config.js --no-daemon",
-        "lint": "eslint . --ext .ts",
-        "lint:fix": "eslint . --ext .ts --fix"
-    }
-}
+  "name": "sudobot",
+  "description": "A Discord bot for moderation purposes.",
+  "version": "5.0.0",
```
Comment (virtual-designer) [inline package.json:4]: Hmm looks like the auto release drafter has made this change here. This should not happen. Please commit this change here.

```suggestion
  "version": "4.0.1",
```

**Comment by virtual-designer** [comment]
> If theres one, people wont have an idea of what some of the commands look like. You can also make the images go side by side and make them smaller. >-<

I totally understand. But if there are so many images, then people would need scroll down to read the stuff which is pretty annoying.
You can combine those images together in one or give me some time, I will do.

**Comment by virtual-designer** [comment]
I've added a "Screenshots" page on the docs. Check it out at https://docs.sudobot.onesoftnet.eu.org/features/screenshots/ 
I think that solves this thing.

**Pull request closed without merging**